Library/ViewModels/ThanksViewModel.swift

```diff
@@ -167,19 +152,6 @@ public final class ThanksViewModel: ThanksViewModelType, ThanksViewModelInputs,
     self.showGamesNewsletterAlert
       .observeValues { AppEnvironment.current.userDefaults.hasSeenGamesNewsletterPrompt = true }
 
-    project
```

Comment:
Seeing as all of this is handled by `SKStoreReviewController.requestReview()` none of this tracking should be needed (nor can it be tracked as far as I can tell).